agent/config/default.go

```diff
@@ -123,6 +123,7 @@ func DefaultSource() Source {
 		telemetry = {
 			metrics_prefix = "consul"
 			filter_default = true
+			prefix_filter = [ "-consul.api.http" ]
```

Comment:
Added it here, please tell me if it should be added elsewhere.

Reply:
I think this should be correct. I'll need to double check. A comment explaining why this is disabled by default would be great. Something about it being a new metric and will be enabled by default in a future version.